qlib/workflow/task/update.py

```diff
@@ -0,0 +1,159 @@
+from typing import Union, List
+from qlib.workflow import R
+from tqdm.auto import tqdm
+from qlib.data import D
+import pandas as pd
+from qlib.utils import init_instance_by_config
+from qlib import get_module_logger
+from qlib.workflow import R
+from qlib.model.trainer import task_train
+from qlib.workflow.recorder import Recorder
+from qlib.workflow.task.collect import TaskCollector
+
+
+class ModelUpdater(TaskCollector):
```

Comment:
The relationship of   ModelUpdater and TaskCollector is weird.
Why we should use subclass here?